docs/_docs/configuration/webrick.md

```diff
@@ -18,3 +18,16 @@ Jekyll provides by default `Content-Type` and `Cache-Control` response
 headers: one dynamic in order to specify the nature of the data being served,
 the other static in order to disable caching so that you don't have to fight
 with Chrome's aggressive caching when you are in development mode.
+
+
+You can override the default `Cache-Control` by setting it to a value of your
+own choosing (a caching instruction, or an empty string). If you don't want the WEBrick server
+to send any `Cache-Control` header at all, set the value to nil. That configuration
+is useful when Jekyll/WEBrick is behind a proxy server, such as `netlify dev`.
+
+```yaml
+# File: _config.yml
+webrick:
+  headers:
+    Cache-Control: nil
```

Comment:
*Attention:* YAML treats this `nil` as a string `"nil"`
You should instead use `null` or `~` or leave undefined / empty.

Reply:
Ouch.  How could I have forgotten that...  And my unit tests explicitly set the value to nil in a hash (which makes perfect sense in Ruby), rather than actually reading from YAML. So my use of nil in the hash worked, but it will not when actually read.

I'm going to go back and do a bit more end-to-end local testing, and then update documentation accordingly.

I'll have an updated pull request this evening (about 12 hours from now in my TZ).